src/api/Dockerfile.frontend-base

```diff
@@ -2,6 +2,8 @@ FROM openbuildservice/base
 
 RUN /root/bin/docker-bootstrap.sh frontend
 
+# necessary for npm install -g
+RUN useradd -r --shell /dev/null nobody
```

Comment:
and don't: just install it. zypper -n in system-user-nobody

Reply:
a great, done